src/pymor/algorithms/timestepping.py

```diff
@@ -228,3 +228,9 @@ def explicit_euler(A, F, U0, t0, t1, nt, mu=None, num_values=None):
                 R.append(U)
 
     return R
+
+
+def depends_on_time(obj, mu):
+    if not mu:
```

Comment:
Do you explicitly want this in the public API?

Reply:
Don't think it hurts. I was actually thinking about adding such a method to `ParametricObject` but decided against it for the time being ..

Reply:
If this is to be a free function I feel it's a weird place for it. I think it belongs with the class def for `ParametricObject`

Reply:
It's only needed in this module so far. Adding it to `ParametricObject` is somewhat problematic, as it suggests to answer whether or not the object depends on time, but the result of this function depends on the given `Mu`. Thus, a different name would be needed. If that makes you happier, I can rename the function to `_depends_on_time` for now.

Reply:
That would indeed make me happier, thanks.